brightness-mode-explainer.md

```diff
@@ -0,0 +1,131 @@
+# Design Document: the need to control screen brightness
+
+_As discussed in the [DAS WG 2021-11-17](https://www.w3.org/2021/11/17-dap-minutes.html) meeting about Screen Brightness and the Screen Wake Lock API, the idea of this document is to make it easier for the meeting attendees to work on an explainer than if this had been committed to GitHub directly. Once this is done the goal is to move the contents to a proper explainer on GitHub._
+
+Previously:
+- https://github.com/WICG/proposals/issues/17
+- https://github.com/w3c/screen-wake-lock/issues/129
+- https://github.com/w3c/devicesensors-wg/issues/51
+- https://www.w3.org/2021/11/17-dap-minutes.html
+
+## Introduction
+
+There is currently a need for web applications to increase the brightness of a device's screen.
+
+This would help address the following use cases:
+- a user scans QR/barcode under various lighting conditions.
+- remote medical, where increasing the screen brightness could assist in remote examination
+- biometric security apps, where increasing the screen brightness can help illuminate features to get better imagery from the front facing camera
+- increasing contrast for readability for the visually impaired
+- [make up mirror](https://play.google.com/store/apps/details?id=mmapps.mirror.pro&hl=en&gl=US) style apps
+
+[Screen Wake Lock API issue 129](https://github.com/w3c/screen-wake-lock/issues/129) was filed back in 2018, and since then there has been feedback from developers working on multiple areas about the usefulness of an API that addresses this use case.
+
```

Comment:
We might want to make a case here that native applications support this, and maybe provide a few examples (e.g., I think Eventbright does this for QR codes, but need to check).

Reply:
That's a very good idea. @willmorgan @kenchris @beaufortfrancois help here would be appreciated :-)

Reply:
I've added a subitem with a few links to the Google Play store for the Starbucks app, the UK Home Office Document Check App and the Dutch government's CoronaCheck app.

I don't have first-hand experience with the Starbucks app so I hope I've linked to the right app and that it actually does what we think it does :-) I believe Will when it comes to the Document Check App, and I've used the Dutch CoronaCheck app myself and know it changes the device's screen brightness level. If anyone has links to other apps (like the Tesco one), please let me know.